ca/docker/confd/templates/ca_config.json.tmpl

```diff
@@ -0,0 +1,40 @@
+{
+    "signing": {
+        "default": {
+            "expiry": "2184h"
+        },
+        "profiles": {
+            "client": {
+                "auth_key": "client",
+                "expiry": "8760h",
+                "name_whitelist": "{{ getv "/opg/ca/clientnamewhitelist" }}",
+                "usages": [
+                    "signing",
+                    "key encipherment",
+                    "client auth"
+                ]
+            },
+            "server": {
+                "auth_key": "server",
+                "expiry": "8760h",
+                "usages": [
+                    "signing",
```

Comment:
Confirmed that server profile usage extensions to the SSL cert allow it to be used for web server use.